js/notify/settings/configure-alert-position.css

```diff
@@ -0,0 +1,143 @@
+html, body {
+    background: #F9F9FA;
+    margin: 0;
+    height: 100%;
+    font-family: SymphonyLato, Lato, serif;
```

Comment:
Please default this to sans-serif. Let the OS decide what sans-serif font to use. Let the opened configuration window have as close to native styles as possible.